english/exercise/seek_veg/seek_veg.md

```diff
@@ -0,0 +1,2 @@
+You go on a quest for vegetables, unfortunately, they are scarce in the land of marshmallows.
+You quest on for many hours over many miles only to find [Althaea officinalis](https://en.wikipedia.org/wiki/Althaea_officinalis). 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.